CONTRIBUTING.md

```diff
@@ -80,12 +80,35 @@ For `static files`, the name pattern is `<module_name>.ext` (i.e.
 ...). Don't link data (image, libraries) outside Odoo: don't use an url to an
 image but copy it in our codebase instead.
 
+### Installation hooks
+When **`pre_init_hook`** and **`post_init_hook`** are used, they should be placed
+in **`init_hooks.py`** located at the root of module directory structure and keys
```

Comment:
> I'm thinking that as uninstall hook is not strictly an "init hook", we should call the file simply hooks.py.

I agree. Also because `init` usually refers to the installation of an addons (i.e. `--init` command line).